lte/gateway/python/BUILD.bazel

```diff
@@ -8,3 +8,46 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+load("@io_bazel_rules_docker//container:container.bzl", "container_image", "container_layer", "container_push")
+load("@io_bazel_rules_docker//docker/package_managers:download_pkgs.bzl", "download_pkgs")
+load("@io_bazel_rules_docker//docker/package_managers:install_pkgs.bzl", "install_pkgs")
+
+config_setting(
+    name = "amd64",
+    constraint_values = ["@platforms//cpu:x86_64"],
+)
+
+config_setting(
+    name = "aarch64",
+    constraint_values = ["@platforms//cpu:arm64"],
+)
+
+download_pkgs(
+    name = "python_interpreters_packages",
+    packages = [
+        "python",  # Python 2 seems to be required during the build
+        "python3",
+        "python3-distutils",  # make strtobool from deprecated distutils work
+        ],
+    image_tar = select({
```

Comment:
why is this also needed in install_pkgs?

Reply:
Both `download_pkgs` and `install_pkgs` execute apt-get inside a docker container. The output of `download_pkgs` is only a `.tar` file containing the downloaded packages, which is passed to `install_pkgs`. But that `.tar` containing the packages file has no information about the docker image, so that needs to be passed to `install_pkgs` as well.